lib/rules/prop-name-casing.js

```diff
@@ -64,6 +65,12 @@ module.exports = {
     schema: [
       {
         enum: allowedCaseOptions
+      },
+      {
+        type: 'object',
+        properties: {
+          ignores: { type: 'array' }
```

Comment:
Please rename the option from `ignores` to `ignoreProps`, so it's clear that you can't specify component names whose props will be ignored.